### ISSUE TITLE
Angular Universal : ReferenceError: Document is not defined

### ISSUE THREAD

**Issue by iwoogy** [themyth92/ngx-lightbox#41]
Hello

Thanks for your component, it has been really useful. 

I'm about to deploy and I found an issue with Angular Universal. 

I got the error ReferenceError: Document is not defined.

![2019-11-14_19-29-39](https://user-images.githubusercontent.com/479162/68857429-54678700-0715-11ea-867f-a9154743a672.jpeg)

![2019-11-14_19-28-47](https://user-images.githubusercontent.com/479162/68857516-84af2580-0715-11ea-80d0-2f34139604ed.jpeg)

I followed the instruction on Angular Universal by adding 
global['window'] = window;
in server.ts

![2019-11-14_19-29-07](https://user-images.githubusercontent.com/479162/68857505-7d881780-0715-11ea-8679-de7fc29af613.jpeg)

![2019-11-14_19-27-30](https://user-images.githubusercontent.com/479162/68857524-8a0c7000-0715-11ea-8233-f375a4787b2b.jpeg)

Thanks for your help

Bobby

**Comment by themyth92** [comment]
Do you [window](https://www.npmjs.com/package/window) lib installed and import already ?

**Comment by iwoogy** [comment]
I upgraded ngx-lightbox to version 2.1.1 from 2.0.0 and I installed window too.

![2019-11-15_17-39-33](https://user-images.githubusercontent.com/479162/68937552-2a729b00-07cf-11ea-805f-6fe0e3eb2041.jpeg)

I required window in server.ts 

![2019-11-15_17-25-03](https://user-images.githubusercontent.com/479162/68937578-36f6f380-07cf-11ea-9db4-f7427f73d4b3.jpeg)

Then I have this error :

![2019-11-15_17-31-24](https://user-images.githubusercontent.com/479162/68937664-6279de00-07cf-11ea-87c6-477127eb9fa8.jpeg)

Do you have an idea how to fix it ?

**Comment by themyth92** [comment]
Hey I dont know whether this error relates to lightbox. Maybe this [issue](https://github.com/webpack/webpack/issues/718) is related ?

**Comment by iwoogy** [comment]
Could be.

I think it's better to avoid dependencies on window for Angular Universal.

Let's close this ticket.

**Comment by themyth92** [comment]
Yeah I agree that using extra dependancy is not really needed. However, if u have any solutions that would not require using `window` dependancy, PR is always welcomed and I would be more than happy to merge it.

**Comment by MehmetSert** [comment]
I encountered the "window" problem. I solved it using "domino".
I share my files, maybe for someone's benefit.

package.json
![Screenshot at Dec 02 02-08-15](https://user-images.githubusercontent.com/16652871/100807429-5c3eba00-3443-11eb-8668-bee9da7ee33c.png)



server.ts
![Screenshot at Dec 02 02-07-55](https://user-images.githubusercontent.com/16652871/100807396-4c26da80-3443-11eb-88c3-48ffde21dade.png)
